resources/lib/vod.py

```diff
@@ -75,8 +96,11 @@ def list_dir(path):
                 'title': asset['title'],
                 'mediatype': 'movie',
                 'originaltitle': asset['original_title'],
-                'plot': asset['synopsis'],
+                #'plot': asset['synopsis'],
```

Comment:
asset.get("synopsis", "") would be a bit nicer perhaps, then you dont need the if clause below.